src/libmongoc/doc/mongoc_database_command.rst

```diff
@@ -17,9 +22,9 @@ Synopsis
                            const bson_t *command,
                            const bson_t *fields,
                            const mongoc_read_prefs_t *read_prefs)
-     BSON_GNUC_WARN_UNUSED_RESULT;
+     BSON_GNUC_WARN_UNUSED_RESULT BSON_GNUC_DEPRECATED_FOR (mongoc_database_command_simple);
```

Comment:
I believe these attribute-like macros should be omitted from documentation (similar to how `MONGOC_EXPORT` is omitted). These control compiler output but are not meant for users to read (these documentation pages serve that purpose instead).